spanner/tests/system/test_system.py

```diff
@@ -843,7 +843,8 @@ def test_read_w_index(self):
         ]
         pool = BurstyPool()
         temp_db = Config.INSTANCE.database(
-            'test_read_w_index', ddl_statements=DDL_STATEMENTS + EXTRA_DDL,
+            'temp_db' + unique_resource_id('_'),
+            ddl_statements=DDL_STATEMENTS + EXTRA_DDL,
```

Comment:
It would be better to redo this part after landing the separate PR which adds the index to the main fixture.